release/staging-resources-cdk/test/GitHubAccessStack.test.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as cdk from 'aws-cdk-lib';
```

Comment:
Let's avoid wildcard imports and be explicit about what classes we are taking dependencies on. You should be able to just do:

```
import { App } from 'aws-cdk-lib';
```

Please update `StagingResourcesStack.test.ts` as well.

Reply:
Thanks for catching that. I was avoiding those, but missed them from the boilerplate code from the tests. I'll fix that.